.gitignore

```diff
@@ -0,0 +1 @@
+/.gitignore
```

Comment:
This means that `.gitignore` is ignored on commit, right?

We should commit `.gitignore` as well, so this file should be empty.

Reply:
Done.